test/information_estimates_test.cpp

```diff
@@ -0,0 +1,97 @@
+
+#ifndef SLAM_TOOLBOX_SLAM_TOOLBOX_INFORMATION_ESTIMATES_TEST_H_
+#define SLAM_TOOLBOX_SLAM_TOOLBOX_INFORMATION_ESTIMATES_TEST_H_
+
+#include <gtest/gtest.h>
+#include "slam_toolbox/experimental/information_estimates.hpp"
+
+
+TEST(UtilsInformationEstimatesTests, SigNumTest)
+{
+    InformationEstimates information_estimates;
+
+    ASSERT_EQ(utils::grid_operations::signum(10), 1) << "FAIL in positive";
+    ASSERT_EQ(utils::grid_operations::signum(-10), -1) << "FAIL in negative";
+    ASSERT_EQ(utils::grid_operations::signum(0), 0) << "FAIL in zero";
+}
+
+TEST(UtilsInformationEstimatesTests, RayCastTest)
+{
+    InformationEstimates information_estimates;
+
+    karto::Vector2<int> initial_pt{22,43};
+    karto::Vector2<int> final_pt{29,38};
+
+    std::vector<karto::Vector2<int>> cells = utils::grid_operations::rayCasting(initial_pt, final_pt);
```

Comment:
@kmilo7204 hmm, why not testing all cells? Also, this is testing ray casting in the fourth quadrant only. May be worth testing the other three.